src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -3783,7 +3783,7 @@ You should consider suppressing the warning only if you're sure that you don't w
     <value>Inconsistent accessibility: event type '{1}' is less accessible than event '{0}'</value>
   </data>
   <data name="ERR_NamedArgumentSpecificationBeforeFixedArgument" xml:space="preserve">
```

Comment:
Yes. I added a note to work items list [https://github.com/dotnet/roslyn/issues/19890](https://github.com/dotnet/roslyn/issues/19890)

---
In reply to: [122046581](https://github.com/dotnet/roslyn/pull/20121#discussion_r122046581) [](ancestors = 122046581)